server/runner/schemasync/syncer.go

```diff
@@ -387,6 +392,12 @@ func (s *Syncer) SyncDatabaseSchema(ctx context.Context, instance *api.Instance,
 		}
 		database = createdDatabase
 	}
+	// Sync database schema
+	if s.profile.Mode == common.ReleaseModeDev {
+		if err := syncDBSchema(ctx, s.store, database, schema); err != nil {
```

Comment:
Should we skip the syncTableSchema/syncViewSchema/syncDBExtensionSchema?

Reply:
Seems not implementation.

Reply:
We are ready not yet, but that's the plan.